web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -251,7 +251,9 @@
 
         int count = 0;
         for (EventCrf evCrf : eventCrfs) {
-            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()) {
+            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()
```

Comment:
Yogi,
As a result of my investigation , there is no import aliasing mechanism in Java. You cannot import two classes with the same name and use both of them unqualified.
Import one class and use the fully qualified name for the other one within the code. You can do aliasing with Groovy. thanks